elevator_move_base_pr2/sample/fetch-sandwich.l

```diff
@@ -0,0 +1,371 @@
+#!/usr/bin/env roseus
+;; -*- coding: utf-8 -*-
+;; fetch-sandwich.l
+;; Author: furushchev <furushchev@jsk.imi.i.u-tokyo.ac.jp>
+
+(ros::roseus-add-msgs "jsk_recognition_msgs")
+(ros::roseus-add-msgs "sensor_msgs")
+(ros::roseus-add-srvs "std_srvs")
+
+(require :pr2-interface "package://pr2eus/pr2-interface.l")
+(require :app-utils "package://jsk_demo_common/euslisp/app-utils.l")
+(require :pr2-action "package://jsk_demo_common/euslisp/pr2-action.l")
+(require :eng2-scene "package://jsk_maps/src/eng2-scene.l")
+
+(load "package://elevator_move_base_pr2/src/utils.l")
+
+
+(defvar *initial-pose*)
+(defvar *pr2*)
+(defvar *ri*)
+(defvar *scene*)
+
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;; PR2 Utility Functions
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+(defun look-front ()
+  (send *pr2* :head :angle-vector #f(0 0))
+  (send *ri* :angle-vector (send *pr2* :angle-vector) 1000)
+  (send *ri* :wait-interpolation))
+
+
+(defun pr2-pick-bag-pose ()
+  #|
+  (send *pr2* :reset-pose)
+  (send *pr2* :torso :waist-z :joint-angle 325.0)
+  (let ((cds (make-cascoords :pos (float-vector 800 0 1300))))
+    (send cds :rotate pi/2 :x)
+    (send *pr2* :rarm :inverse-kinematics
+          cds
+          :use-torso t
+          :look-at-target t))
+  |#
+  (float-vector 325.0 60.0 74.0 70.0 -120.0 20.0 -30.0 180.0 -13.5232 -15.8808
+                -82.6112 -83.1698 -162.008 -72.0918 170.603 0.0 40.2852))
+
+
+(defun wait-for-hand-over (arm &optional (timeout 10) (n-trial 3))
+  (dotimes (i n-trial)
+    (when (wait-for-hand-impact arm :timeout timeout)
+      (ros::ros-info "Received sandwich.")
+      (speak-jp "ありがとうございます。" :wait t)
+      (return-from wait-for-hand-over t))
+    (ros::ros-warn
+     (format nil "Hand impact timed out (~A sec) without response" timeout))
+    (speak-jp "袋を私の手に掛けて、手を揺らしてください。" :wait t))
+  ;; fail
+  (speak-jp "あれ、サンドイッチ。。。" :wait t)
+  (return-from wait-for-hand-over nil))
+
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;; Actions
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+
+(defun init ()
+  (setq *pr2* (pr2))
```

Comment:
we need to set pr2 robot name

```suggestion
  (setq *pr2* (pr2 :pr1040))
```

Reply:
@tkmtnt7000 @mqcmd196 
I think we need this change to #1397 .

Reply:
Nice comment and thank you for your information @pazeshun @tkmtnt7000

Reply:
fixed https://github.com/jsk-ros-pkg/jsk_demos/pull/1397/commits/2338f3ab52e537fefb91b76f0c825a20684288d3

Reply:
Hmm, but I think the better solution is setting with args or something

Reply:
Something like https://github.com/jsk-ros-pkg/jsk_robot/blob/23037dbfe66757052074a8109ab61875a73c3a4c/jsk_fetch_robot/jsk_fetch_startup/euslisp/navigation-utils.l#L19-L26 ??